doc/source/getting_started/how_to_read_these_docs.rst

```diff
@@ -0,0 +1,41 @@
+.. _howtoread:
+
+{{ header }}
+
+**********************
+How to read these docs
+**********************
+
+In these tutorials you will see input code inside code blocks such as:
+
+::
+
+    import pandas as pd
+    pd.DataFrame({'A': [1, 2, 3]})
+
+
+or:
+
+.. ipython:: python
+
+    import pandas as pd
+    pd.DataFrame({'A': [1, 2, 3]})
+
+
+
+The first block is a standard python input, while in the second the ``In [1]:`` indicates the input is inside a `notebook <https://jupyter.org>`__. It does not make any difference **except the last line will be printed**.
```

Comment:
Hi @rhuille - I'm not sure about this "except the last line will be printed" part. Is that really a defining feature of Jupyter Notebooks? Even in a Python repl, you would see the last line printed. Perhaps mention that, in Jupyter Notebooks, plots will be shown inline (e.g. see #39527 where someone was confused by how to work with the visualisation docs)

Reply:
Hi @MarcoGorelli 
Thanks for your feedback, I agree it's not defining Jupyter Notebooks, so let's rewrite it like this: https://github.com/pandas-dev/pandas/pull/39207/commits/27c6bc6dc182f22c03f51b132b1259f3594ac8fa what do you think ?